bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -4326,6 +4331,25 @@ String addSourceToArtifactAttribute(BlackboardAttribute attr, String source) thr
 	 *                          within tsk core
 	 */
 	public BlackboardArtifact.Type addBlackboardArtifactType(String artifactTypeName, String displayName) throws TskCoreException, TskDataException {
+		
+		return addBlackboardArtifactType(artifactTypeName, displayName, BlackboardArtifact.CategoryType.EXTRACTED_DATA);
+	}
+
+	/**
+	 * Add an artifact type with the given name and category. Will return an artifact Type.
+	 *
+	 * @param artifactTypeName System (unique) name of artifact
+	 * @param displayName      Display (non-unique) name of artifact
+	 * @param category		   Artifact type category.
+	 * 
+	 *
+	 * @return Type of the artifact added.
+	 *
+	 * @throws TskCoreException exception thrown if a critical error occurs
+	 * @throws TskDataException exception thrown if given data is already in db
+	 *                          within tsk core
+	 */
+	public BlackboardArtifact.Type addBlackboardArtifactType(String artifactTypeName, String displayName, BlackboardArtifact.CategoryType category) throws TskCoreException, TskDataException {
```

Comment:
We should not add new artifact-related methods to the public API of SleuthkitCase.  See email for discussion. This method should only be exposed via Blackboard.

Reply:
@rcordovano : I don't quite follow this comment.   Could you please elaborate?

 addBlackboardArtifactType() is already a method in SleuthKitCase.  I did add an overloaded variant that takes in the Category as well (and modified existing overload to call the other with category EXTRACTED_DATA by default.